### DISCUSSION TITLE
discussion: cause form factor

### DISCUSSION THREAD

**Discussion by thelostone-mc** [gitcoinco/grants-stack#957] [Q&A]
### Description

Read the thread over at https://community.supermodular.xyz/t/new-form-factors-for-quadratic-funding/165/3?u=thelostone-mc

The TLDR is exploring/brainstorming to see what changes we may have to make to see if we wanted to add a feature to allow curation of projects within a grant round and curators get incentivized to help make the donation to projects 1 step quicker.
Thinking out this would require no contract changes unless we want to create some sort of curator score but I'm not sure what that might look like.

For the grant explorer: 
- Allows + Incentivizes curation giving round operators more insights
- Folks might like to donate by cause as opposed to adding every grant to shortlist  
- Potentially creates more engagement as curators get reward 

Only downside I can think of is that curator in this sense is short-lived AKA for every round it would have to be redone 
The upside of that is curators can choose which round they would like to curate 
We can further make this exciting by allowing round creators to decide if would want to allow curation within a round / add in some means of encouraging curators to come make their round more engaging  

For the grant hub: 
- Might be an interesting data point to see how many times a project has been included in a curation piece.
